CONTRIBUTING.md

```diff
@@ -15,79 +15,78 @@ highly appreciated!
   > may cause all kinds of chaos in your development environment.
 
-- Open the Extension Development Host, a child instance of VS Code with the
-  results of the dev servers installed as an extension, with `Start Debugging`
-  (f5).
+- **Start Debugging** (F5) to open the [Extension Development Host], a child
+  instance of VS Code with the local extension build installed.
 
   > **Note**: using the `Run Extension` command when running the debugger will
   > prevent all other extensions from loading into VS Code. This will improve
   > the performance of VS Code but can cause certain DVC commands to fail if the
-  > project uses an isolated python environment (see [this warning](#warning)).
+  > DVC project uses an isolated Python env (see [this warning](#warning)).
 
-- Open a DVC project in the Extension Development Host. VS Code will remember
+- Open a DVC project in the _Extension Development Host_. VS Code will remember
   the last project opened, so this step only has to be done once.
 
   > **Note**: We have provided a demo project as part of this repo, but feel
   > free to use any DVC project that you have available.
 
-## The demo project
+[extension development host]:
+  https://code.visualstudio.com/api/working-with-extensions/testing-extension
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+## The demo project
 
-- Go to the `demo/` directory of this repo.
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+your changes to this extension.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+> **Note**: It is not an exhaustive showcase of DVC's features. Testers are
+> encouraged to try other DVC repositories -- especially real-world cases!
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Run `yarn setup:venv` from the _Terminal_ in the root of this repo to set up a
+  Python virtual environment for the demo project (in `demo/.env`).
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- Open the `./demo` project in a VS Code window, for example in the _Extension
+  Development Host_ (see previous section).
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- Pull the project data using this extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** in _Source Control_) or by
+  running `dvc pull` from _Terminal_.
```

Comment:
from the _Integrated Terminal_?

Reply:
https://code.visualstudio.com/docs/editor/integrated-terminal

> If you use a vs code terminal at this point, we know you're in the project root.